components/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,121 @@
+require('./Carousel.scss')
+import classNames from 'classnames'
+
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+
+import LeftArrowIcon from '../Icons/LeftArrowIcon'
+import RightArrowIcon from '../Icons/RightArrowIcon'
+
+export default class Carousel extends Component {
+  componentWillMount() {
+    this.handleResize = this.handleResize.bind(this)
+    window.addEventListener('resize', this.handleResize)
+    this.handlePageUp = this.handlePageUp.bind(this)
+    this.handlePageDown = this.handlePageDown.bind(this)
+    this.setState({firstVisibleItem: this.props.firstVisibleItem || 0})
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.handleResize)
+  }
+
+  handleResize() {
+    this.validatePagers()
+  }
+
+  componentDidMount() {
+    this.validatePagers()    
+  }
+
+  componentDidUpdate() {
+    this.validatePagers()
+  }
+
+  validatePagers() {
+    const pageDownClass = classNames({
```

Comment:
You can comma separate the arguments, so for something like a class that always needs to be added you can do:
```
classNames(
  'page-down',
  { hidden: this.state.firstVisibleItem === 0 }
)
```